kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/Filterable.java

```diff
@@ -38,5 +42,7 @@
   T withFields(Map<String, String> labels);
 
   T withField(String key, String value);
+
+  T withSelector(LabelSelector selector);
```

Comment:
Can we call it `withLabelSelector` please, just to be consistent with other method names?